worker/transcribee_worker/torchaudio_align.py

```diff
@@ -257,7 +262,11 @@ def work(queue, audio):
                     waveform_segment.size(1) / (trellis.size(0) - 1)
                 ) / settings.SAMPLE_RATE
                 for i, atom in enumerate(atoms):
-                    (start, last_end), (end, next_start) = atom_index_to_timing_index[i]
+                    start_info, end_info = atom_index_to_timing_index[i]
+                    if start_info is None or end_info is None:
+                        continue
```

Comment:
Could you please add a comment why/how this can happen?

Reply:
This is quite a quick a dirty fix for now. I started refactoring the code, but actually I think we should completely redo the alignment instead